examples/t1.qasm

```diff
@@ -12,7 +12,7 @@ int[32] counts0, counts1 = 0;   // surviving |1> populations of qubits
 
 kernel tabulate(int[32]:counts, int[32]:shots, int[32]:num);
 
-cbit c0, c1;
+bit c0, c1;
 
 // define a gate calibration for an X gate on qubit 1
 defcal "openpulse" x %0 {
```

Comment:
For `reset %0` below (line 25)--is a qubit register required? I am guessing it is not as we are referencing the physical qubit of interest.